src/main/scala/org/hatdex/hat/api/json/HatJsonFormats.scala

```diff
@@ -88,12 +88,12 @@ trait HatJsonFormats extends HatJsonUtilities with UuidMarshalling with LocalDat
     unlift(ApiBundleDataSourceField.unapply)
   )
 
-  implicit val ApiBundleDataSourceDatasetFormat   = Json.format[ApiBundleDataSourceDataset]
-  implicit val ApiBundleDataSourceStructureFormat = Json.format[ApiBundleDataSourceStructure]
-  implicit val ApiBundleContextlessFormat         = Json.format[ApiBundleContextless]
+  implicit val ApiBundleDataSourceDatasetFormat: OFormat[ApiBundleDataSourceDataset]   = Json.format[ApiBundleDataSourceDataset]
+  implicit val ApiBundleDataSourceStructureFormat: OFormat[ApiBundleDataSourceStructure] = Json.format[ApiBundleDataSourceStructure]
+  implicit val ApiBundleContextlessFormat: OFormat[ApiBundleContextless]         = Json.format[ApiBundleContextless]
```

Comment:
And here

Reply:
Pretty sure that scalafix added the return type.  That impacts the formatting.

We don't have a Scalafmt rule configured to align on `=` but we can.